File: thorium/src/test/integration/add/testAdd.R
csv <- "data.csv"
table <- read.csv(csv, stringsAsFactors = FALSE)
table[nrow(table) + 1,] <- c("pierre",21,"Rennes","GMA")
rows <- list(c("serge",21,"Rennes","GMA"),c("paul",22,"Paris","GMA"),c("herve",23,"Lyon","INFO"))
for (row in rows){
	table[nrow(table) + 1,] <- row
}
write.csv(table, "./src/test/integration/add/generated.csv", row.names=FALSE, quote=FALSE)

File: thorium/src/test/integration/add/testAdd.py
import pandas as pd
csv = "data.csv"
table = pd.read_csv(csv)
values = "pierre,21,Rennes,GMA".split(",")
new_row = pd.DataFrame([values],columns=table.columns)
table = pd.concat([table,new_row], ignore_index=True)
new_values = ["serge,21,Rennes,GMA","paul,22,Paris,GMA","herve,23,Lyon,INFO"]
for row in new_values:
	values = row.split(",")
	new_row = pd.DataFrame([values],columns=table.columns)
	table = pd.concat([table,new_row], ignore_index=True)
table.to_csv("./src/test/integration/add/generated.csv",index=False)
